pkg/apis/kappctrl/v1alpha1/types_fetch.go

```diff
@@ -86,3 +87,14 @@ type AppFetchLocalRef struct {
 	// Object is expected to be within same namespace
 	corev1.LocalObjectReference `json:",inline" protobuf:"bytes,1,opt,name=localObjectReference"`
 }
+
+type AppFetchImgpkgBundle struct {
+	Image string `json:"image,omitempty"`
+	// Secret may include one or more keys: username, password, token.
+	// By default anonymous access is used for authentication.
+	// TODO support docker config formated secret
+	// +optional
+	SecretRef *AppFetchLocalRef `json:"secretRef,omitempty"`
+	// +optional
+	SubPath string `json:"subPath,omitempty"`
```

Comment:
i dont think we want to provide subPath for imgpkgBundle because there wouldnt be a way to access .imgpkg. (could always add later if we find a need for this).

Reply:
Makes sense. Will remove.

Reply:
Removed.